tools/devctr/Dockerfile.x86_64

```diff
@@ -52,7 +52,9 @@ RUN apt-get update \
         wheel \
     && python3 -m pip install \
         boto3 \
+        dataclasses \
```

Comment:
The commit description mentions only `psutil` being needed, but is there a reason for adding `dataclasses` and `typing-extensions`?

Reply:
I've added them to keep track of changes added for v18 dev container, added in #2051. We should merge first #2051, and then adjust this PR by pinning support for network latency test.